github-flow/update_github_default_branch.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+import json
+import os
+
+import requests
+
+access_token = os.environ.get("GITHUB_ACCESS_TOKEN")
+
+
+def update_github_default_branch(branch_name):
+    headers = {"Authorization": f"token {access_token}", "Content-Type": "application/json"}
+    data = {"name": "knowledge-engine", "default_branch": branch_name}
+    url = "https://api.github.com/repos/CognitionX/knowledge-engine"
```

Comment:
Would it be nice to specify the repo or will this just be used for the knowledge-engine?